cluster/vagrant/util.sh

```diff
@@ -33,13 +33,62 @@ function detect-minions {
 # Verify prereqs on host machine  Also sets exports USING_KUBE_SCRIPTS=true so
 # that our Vagrantfile doesn't error out.
 function verify-prereqs {
-  for x in vagrant VBoxManage; do
+  for x in vagrant; do
     if ! which "$x" >/dev/null; then
       echo "Can't find $x in PATH, please fix and retry."
       exit 1
     fi
   done
 
+  local vagrant_plugins=$(vagrant plugin list | sed '-es% .*$%%' '-es%  *% %g' | tr ' ' $'\n')
+  local providers=(
+      # Format is:
+      #   provider_ctl_executable vagrant_provider_name vagrant_provider_plugin_re
+      # either provider_ctl_executable or vagrant_provider_plugin_re can
+      # be blank (i.e., '') if none is needed by Vagrant (see, e.g.,
+      # virtualbox entry)
+      vmrun vmware_fusion vagrant-vmware-fusion
+      vmrun vmware_workstation vagrant-vmware-workstation
+      prlctl parallels vagrant-parallels
+      VBoxManage virtualbox ''
+  )
+  local provider_found=''
+  local provider_bin
+  local provider_name
+  local provider_plugin_re
+
+  while [ "${#providers[@]}" -gt 0 ]; do
+    provider_bin=${providers[0]}
+    provider_name=${providers[1]}
+    provider_plugin_re=${providers[2]}
+    providers=("${providers[@]:3}")
+
+    # If the provider is explicitly set, look only for that provider
+    if [ -n "${VAGRANT_DEFAULT_PROVIDER:-}" ] \
+        && [ "${VAGRANT_DEFAULT_PROVIDER}" != "${provider_name}" ]; then
+      continue
+    fi
+
+    if ([ -z "${provider_bin}" ] \
+          || which "${provider_bin}" >/dev/null) \
```

Comment:
Whoops! :open_mouth: Thanks for catching that! Believe it or not, I had intended to do that originally. This is just not my week. :disappointed:

I noticed that elsewhere in the file, the older `>/dev/null 2>&1` is used (even though `&>` is supported as of `bash` 4). I followed the existing convention, but can change it if desired.